### PR TITLE
Prepare release artifact WF

### PR DESCRIPTION
**Pull request by R4tmax**
Prepares an outline for how to created automated packages. 
TAKE NOTE: HEAVY TESTING REQUIRED FOR VALIDATION ON END MACHINES.

### CODE REVIEW

**Comment by R4tmax** [comment]
Converting to draft, noticed a couple of errors in the execution logic. Will fix later in the weekend

**Comment by R4tmax** [comment]
Note: follows logic outlined at: https://cx-freeze.readthedocs.io/en/stable/builtdist.html 

Testing will be required as I suspect there might be some execution issues

File: .github/workflows/build_packages.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install cx_Freeze pyinstaller
```
Comment (Teg3z) [inline .github/workflows/build_packages.yml:24]: Do we need to install pyinstaller? I dont see its usage anywhere.

**Review by Teg3z — APPROVED** [review]
Seems great overall, although I added one question in the code.

File: .github/workflows/build_packages.yml (hunk 1 above)
Comment (R4tmax) [reply .github/workflows/build_packages.yml:24]: Nope, i dont think so, forgot to clean it up from the initial draft

**Review by Teg3z — APPROVED** [review]

**Pull request merged**